ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
+}
+
+.carousel img {
+    background-color: rgb(236, 236, 236);
+    height: 70%;
```

Comment:
Sometimes you have px sometimes you have %. Not good. Choose one

Reply:
If i choose % at .galery somehow the divs size doesn't change 20% above, not matter what % I put, from 21% to 100% stays the same

Reply:
because probably there is no more room to grow